src/reactapp/src/context/Form/CheckoutFormProvider.jsx

```diff
@@ -57,6 +58,62 @@ function CheckoutFormProvider({ children }) {
   const { placeOrder } = useCartContext();
   const { setPageLoader } = useAppContext();
 
+  /**
+   * Init value of checkout-form-formik
+   *
+   * It will be the combined object of each individual form sections which
+   * are registered to this provider.
+   *
+   * So the whole initValues would be represented like:
+   * {
+   *    [form_section_id]: { ...form_section_init_values},
+   *    [form_section_id]: { ...form_section_init_values},
+   * }
+   */
+  const formInitialValues = useMemo(
+    () => prepareFormInitValues(sections),
+    [sections]
+  );
+
+  /**
+   * ValidationSchema of checkout-form-formik
+   *
+   * If there is no activeForm, then the validationSchema equals to the combined
+   * validationSchema of each individual form section which are registered with
+   * this provider
+   *
+   * If there is a valid activeForm, then the validationSchema represent the
+   * validationSchema of the active form section.
+   */
+  const formValidationSchema = useMemo(
+    () => prepareFormValidationSchema(sections),
+    [sections]
+  );
+
+  const formSubmit = useCallback(
+    async (values) => {
+      try {
+        setPageLoader(true);
+        const order = await placeOrder(values, paymentActionList);
+
+        const orderNumber = _get(order, 'order_number');
+
+        if (orderNumber && config.isProductionMode) {
+          LocalStorage.clearCheckoutStorage();
+          window.location.replace(config.successPageRedirectUrl);
+        }
+
+        if (orderNumber && config.isDevelopmentMode) {
+          LocalStorage.clearCheckoutStorage();
+        }
+        setPageLoader(false);
+      } catch (error) {
+        setPageLoader(false);
```

Comment:
Should there be anything done with the error? Maybe a `console.error` at least?

Reply:
Not relevant. we are not relying on the above submit action of formik. We have a `PlaceOrder` component which exclusively use its own submit function to place the order. We can even remove this submithandler. But, then i thought to preserve it just in case some one needs it :)

Reply:
But, for the time being, a console.error can be added 👍

Reply:
If the error is handled somewhere else, I would not also add a console entry.